.gitignore

```diff
@@ -65,3 +65,4 @@ target/
 pyuvdata/data/
 pyuvdata/data/test/
 pyuvdata/GIT_INFO
+uvdata/data/test/*
```

Comment:
I added it to prevent files from getting automatically added (github desktop tries to automatically track files) added when I run nosetests locally.